helix-lsp/src/lib.rs

```diff
@@ -1,9 +1,11 @@
 mod client;
+pub mod file_event;
 pub mod jsonrpc;
 pub mod snippet;
 mod transport;
 
 pub use client::Client;
+use file_event::Handler;
```

Comment:
Small readability nit: rather than `use`ing these here, let's qualify the uses of `Handler` since its name is so generic.

Reply:
I agree, fixed!